testing/scripts/test_helpdesk_pw.py

```
@@ -195,8 +196,71 @@ def skip(name: str, reason: str = ""):
+    return []
+
+
+def mailhog_clear() -> None:
+    """Delete all messages from MailHog (no-op when MAILHOG_URL is unset)."""
+    if not MAILHOG_URL:
+        return
+    import requests as _req
+    try:
+        _req.delete(f"{MAILHOG_URL}/api/v1/messages", timeout=5)
+    except Exception:
+        pass
```

Ruff (0.15.10)

[warning] Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(Exception): ...`

(SIM105)